Wishlist dropin (#347)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/containers/wishlist.mdx

---
title: Wishlist container
description: Learn about the Wishlist container.
---

import Diagram from '@components/Diagram.astro';
import OptionsTable from '@components/OptionsTable.astro';

> The Wishlist container manages and displays a list of products wishlist by a user.

Reviewer: ```suggestion
This container manages and displays the list of products on a user's wishlist.
```

> Its behavior is driven by configuration options such as routeEmptyWishlistCTA, routeEmptyWishlistCTA and routing via routeProdDetailPage callbacks.

Reviewer: ```suggestion
Configuration options such as `routeEmptyWishlistCTA`, `routeEmptyWishlistCTA` and routing via `routeProdDetailPage` callbacks drive the container's behavior.
```
